### ISSUE TITLE
Allow for initial position other than 0 in useSlider

### ISSUE THREAD

**Issue by rockwotj** [skozer/react-instagram-zoom-slider#8]
## 🚀 Feature Proposal

Allow for useSlider hook to specify an initial position and update the slide programatically.

## Motivation

I'm building a lightbox with pinch to zoom and I want to use this library but need to open a photo at any position and also need to update the position programatically.

## Example

```
const [zooming, currentSlide, changeSlide, bind, x, onScale] = useSlider({                                                            
  numSlides: previews.length,                                                                                            
  initialSlide: initialIndex,                                                                                 
})
useKey('ArrowRight', () => changeSlide(1), [changeSlide]);
useKey('ArrowLeft', () => changeSlide(-1), [changeSlide]);
```

**Comment by skozer** [comment]
Great suggestion @rockwotj!

From an API perspective, I think the `changeSlide` function should accept either the requested slide index, or a nested function with the current slide index as an argument.

e.g.

```js
changeSlide(3) // go to slide #4
changeSlide((currentSlide) => currentSlide + 1)) // go to slide #5
```

What do you think?

**Comment by rockwotj** [comment]
Sounds good. I'll try and throw together a PR at some point.

**Comment by skozer** [comment]
@rockwotj I've split the `changeSlide` feature into another issue (#13) as @muhraff is working on a PR for `initialSlide`.

Will hopefully get this one merged in soon!

**Comment by skozer** [comment]
Released in 1.4.0
